tox.ini

```diff
@@ -9,18 +9,23 @@ skipsdist = true
 envlist =
     py3
     black
-    flake8 >= 3.7.0
+    flake8
     docs
-    isort >= 5.1.0
+    isort
 
-# Add environment to use the default python3 installation
+# Add environment to use the minmum supported python3 installation
 [testenv:py3]
-basepython = python3
+basepython = python3.8
 
 [testenv]
 usedevelop = true
 # OBJC_DISABLE_INITIALIZE_FORK_SAFETY fixes hung tests for MacOS users. (#30806)
-passenv = DJANGO_SETTINGS_MODULE PYTHONPATH HOME DISPLAY OBJC_DISABLE_INITIALIZE_FORK_SAFETY
+passenv =
+    DISPLAY
+    DJANGO_SETTINGS_MODULE
+    HOME
+    OBJC_DISABLE_INITIALIZE_FORK_SAFETY
+    PYTHONPATH
```

Comment:
This change is unrelated, please revert.

Reply:
Okay, I'll create another PR with only this change then.